client/blocks/comments/post-comment-list.jsx

```diff
@@ -169,6 +240,13 @@ class PostCommentList extends React.Component {
 		);
 	};
 
+	scrollToComment = () => {
+		const comment = window.document.getElementsByName( window.location.hash.substring( 1 ) )[ 0 ];
+		comment.scrollIntoView();
```

Comment:
this makes me a little nervous, but i'm not confident that `scrollIntoView` is a sync method. Are we guarenteed that the scroll is complete when the method returns?